test-data/pretty_format_yaml/multi-doc-pretty-formatted.yaml

```diff
@@ -0,0 +1,17 @@
+---
+root:
+  test:
+  - hi there
+---
+second_root:
+  test:
+  - 1
+  - 2
+---
+This is a primitive file.
+
+Last line of file.
```

Comment:
I would put two consecutive text lines as well.
Suggesting this because on https://repl.it/@macisamuele/language-formatters-pre-commit-hooks-PR3 I've noticed that
```
aaa
bbb

ccc
```
would be read as `'aaa bbb\nccc'`

Reply:
Sure, but it works fine. The reason you had a problem is that you used `load_all()`.